src/components/styles/join.scss

```diff
@@ -106,4 +102,53 @@
       }
     }
   }
+
+  .quotes {
+    display: block;
+    height: fit-content;
+    min-height: 500px;
+    overflow: hidden;
+  }
+
+  .react-3d-carousel .slider-container .slider-right div {
+    border: none;
+  }
+
+  .react-3d-carousel .slider-container .slider-left div {
+    border: 100vw;
+  }
```

Comment:
```suggestion

```
Do not style the carousel from this css, also these classes no longer exist so this is doing nothing.